app/views/layouts/_emails.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if user.additional_email.blank? %>
+  <%= mail_to user.email, user.email %>
+<% else %>
+  <%= mail_to user.additional_email, user.additional_email %>
```

Comment:
I am convinced that the resulting changes in the code became more elegant and logical, it is also a point in your favor =)